platform/android/export/export_plugin.cpp

```diff
@@ -437,6 +437,14 @@ String EditorExportPlatformAndroid::get_project_name(const String &p_name) const
 
 String EditorExportPlatformAndroid::get_package_name(const String &p_package) const {
 	String pname = p_package;
+	String name = get_valid_basename();
+	pname = pname.replace("$genname", name);
+	return pname;
+}
+
+//returns the project name without invalid characters
+//or the "noname" string if all characters are invalid
```

Comment:
Our comment style is full sentences (with capital letter and period), and space after `//`.
```suggestion
// Returns the project name without invalid characters
// or the "noname" string if all characters are invalid.
```
Applies to the other comments below.